application/uitests/org.openjdk.jmc.flightrecorder.uitest/src/test/java/org/openjdk/jmc/flightrecorder/uitest/pages/FileIOPageTest.java

```diff
@@ -53,15 +53,9 @@
 	private static final String WRITE_COUNT_COL = JdkAggregators.FILE_WRITE_COUNT.getName();
 	private static final String RECORDING = "io_test.jfr";
 
-	private static final long[][] TABLE_VALUES = {
-		{  14560, 9212, 11848, 4653 },
-		{  14567, 9212, 41055, 3533 },
-		{ 251007,  223, 41055, 3533 },
-		{ 251007,  223, 43519,    6 },
-		{ 251007,  223, 43519,    6 },
-		{ 251007,  223, 43519,    6 },
-		{ 251007,  223, 43519,    6 },
-	};
+	private static final long[][] TABLE_VALUES = {{14560, 9212, 11848, 4653}, {14567, 9212, 41055, 3533},
+			{251007, 223, 41055, 3533}, {251007, 223, 43519, 6}, {251007, 223, 43519, 6}, {251007, 223, 43519, 6},
+			{251007, 223, 43519, 6},};
```

Comment:
Here I'd definitely like to keep the old structure.

Reply:
Yep.